eogrow/utils/validators.py

```diff
@@ -18,6 +18,16 @@
     from ..core.schemas import ManagerSchema
 
 
+def list_factory() -> List[Any]:
+    """Required due to changes in pydantic mypy-plugin. Hopefully can be replaced by `list` in future."""
+    return []
+
+
+def dict_factory() -> Dict[Any, Any]:
+    """Required due to changes in pydantic mypy-plugin. Hopefully can be replaced by `dict` in future."""
+    return {}
+
+
```

Comment:
If I understand correctly, you did it like this so that in future in can be replaced in a single place, right?

Reply:
if this is ever corrected in mypy or pydantic-mypy plugin, then we can delete this and search-replace all occurrences. Also `list_factory` returns `List[Any]` which can then be specialized to whatever type, however `list` returns `List[_T]` and neither mypy nor the pydantic plugin carry out the typevar instantiation, so it fails. Bizzare days we live in

Reply:
0_0